### PR TITLE
Remove `create_function` call as it is deprecated in 7.3

### PR DESCRIPTION
**Pull request by desyncr**

### CODE REVIEW

**Comment by desyncr** [comment]
Is there anything holding this up?

**Comment by alquerci** [comment]
@desyncr Nothing related directly to this PR but more about the status of @friendsofsymfony1 organisation.

See FriendsOfSymfony1/symfony1#201

cc @thePanz

File: lib/Doctrine/Parser/sfYaml/sfYamlInline.php [hunk 1]
```diff
@@ -135,7 +135,7 @@ static protected function dumpArray($value)
     if (
       (1 == count($keys) && '0' == $keys[0])
       ||
-      (count($keys) > 1 && array_reduce($keys, create_function('$v,$w', 'return (integer) $v + $w;'), 0) == count($keys) * (count($keys) - 1) / 2))
+      (count($keys) > 1 && array_reduce($keys, function ($v, $w) { return (integer) $v + (integer) $w;}, 0) == count($keys) * (count($keys) - 1) / 2))
```
Comment (thePanz) [inline lib/Doctrine/Parser/sfYaml/sfYamlInline.php]: Please use `return (int)($v + $w);

Comment (desyncr) [reply lib/Doctrine/Parser/sfYaml/sfYamlInline.php]: Sure thing.

**Comment by desyncr** [comment]
True. My bad. I could add a php version check at runtime to determine which one to use.

Maybe related: https://github.com/FriendsOfSymfony1/doctrine1/issues/66.

**Comment by thePanz** [comment]
@endelwar @desyncr I created https://github.com/FriendsOfSymfony1/doctrine1/pull/72 to support php 5.3 onwards

**Pull request merged**